cmd/kubeadm/app/phases/addons/dns/manifests.go

```diff
@@ -128,11 +132,6 @@ spec:
             scheme: HTTP
         securityContext:
           allowPrivilegeEscalation: false
-          capabilities:
```

Comment:
This container now does not need any capabilities they can all be dropped:
```
securityContext:
  capabilities:
    drop:
    - ALL
```

Reply:
Updated.